src/api_module.js

```diff
@@ -41,12 +41,12 @@ export class ApiModule {
 
     if (routes.includes("read")) {
       this.read = async function (auth, id, retryOptions = {}) {
-        const request = new Request(`${this.path}/${id}`, {
+        const request = new Request(`${this.path}/${id || ""}`, {
```

Comment:
This doesn't seem related? I also don't think we should want to support passing an empty id?

Reply:
this slipped in, removed from this PR